code/controllers/subsystem/dynamic/dynamic.dm

```diff
@@ -393,16 +393,16 @@ SUBSYSTEM_DEF(dynamic)
 
 	switch(round(shown_threat))
 		if(0 to 19)
-			var/show_core_territory = (GLOB.current_living_antags.len > 0)
-			if (prob(FAKE_GREENSHIFT_FORM_CHANCE))
-				show_core_territory = !show_core_territory
-
-			if (show_core_territory)
-				advisory_string += "Advisory Level: <b>Blue Star</b></center><BR>"
-				advisory_string += "Your sector's advisory level is Blue Star. At this threat advisory, the risk of attacks on Nanotrasen assets within the sector is minor but cannot be ruled out entirely. Remain vigilant."
-			else
-				advisory_string += "Advisory Level: <b>Green Star</b></center><BR>"
-				advisory_string += "Your sector's advisory level is Green Star. Surveillance information shows no credible threats to Nanotrasen assets within the Spinward Sector at this time. As always, the Department of Intelligence advises maintaining vigilance against potential threats, regardless of a lack of known threats."
+			switch(rand(1,3))
+				if(1)
+					advisory_string += "Advisory Level: <b>Yellow Star</b></center><BR>"
+					advisory_string += "Your sector's advisory level is Yellow Star. Surveillance shows a credible risk of enemy attack against our assets in the Spinward Sector. We advise a heightened level of security alongside maintaining vigilance against potential threats."
+				if(2)
+					advisory_string += "Advisory Level: <b>Orange Star</b></center><BR>"
+					advisory_string += "Your sector's advisory level is Orange Star. Upon reviewing your sector's intelligence, the Department has determined that the risk of enemy activity is moderate to severe. At this advisory, we recommend maintaining a higher degree of security and reviewing red alert protocols with command and the crew."
+				if(3)
+					advisory_string += "Advisory Level: <b>Red Star</b></center><BR>"
+					advisory_string += "Your sector's advisory level is Red Star. The Department of Intelligence has decrypted Cybersun communications suggesting a high likelihood of attacks on Nanotrasen assets within the Spinward Sector. Stations in the region are advised to remain highly vigilant for signs of enemy activity and to be on high alert."
```

Comment:
Rather than literally copy pasting the yellow, orange, and red star announcements: just put in some code before the switch:

```dm
if(shown_thread <= 19)
    shown_threat = rand(21, 79)
```

Reply:
Ah, good idea. I'll do that.

Reply:
will this run the switch statement again? last time i was messing with one I was told it doesn't?

Reply:
> will this run the switch statement again? last time i was messing with one I was told it doesn't?

I just committed this change, read the code again. It worked fine when I tested it so we should be good.